services/web/server/src/simcore_service_webserver/projects/projects_nodes_handlers.py

```diff
@@ -116,6 +116,21 @@ async def post_retrieve(request: web.Request) -> web.Response:
     )
 
 
+@routes.post(f"/{VTAG}/projects/{{project_uuid}}/nodes/{{node_uuid}}/restart")
+@login_required
+@permission_required("project.node.read")
+async def post_restart(request: web.Request) -> web.Response:
+    try:
+        node_uuid = request.match_info["node_id"]
+        data = await request.json()
+    except KeyError as err:
+        raise web.HTTPBadRequest(reason=f"Invalid request parameter {err}") from err
+
+    await director_v2_api.restart(request.app, node_uuid)
+
+    return web.HTTPNoContent()
+
+
 @login_required
 @permission_required("project.node.delete")
 async def delete_node(request: web.Request) -> web.Response:
```

Comment:
but I guess the route above was supposed to be used here. Please can you check since when this was broken and what implications it has? that means that the nodes were not properly deleted

Reply:
I think @pcrespov  answers above to your question as well